sea.js

```diff
@@ -96,7 +99,7 @@
     Object.assign(SafeBuffer, {
       // (data, enc) where typeof data === 'string' then enc === 'utf8'|'hex'|'base64'
       from() {
-        if (!Object.keys(arguments).length) {
+        if (!Object.keys(arguments).length || arguments[0]==null) {
```

Comment:
Are you sure that `==' is right comparision to figure out `null` or missing value ? In my past, better stick with more strict comparison `===` and/or have more OR comparisions to figure out all possible error cases.

Reply:
yes just in the case of ==null this will match also undefined.